rllib/models/distributions.py

```diff
@@ -122,3 +126,41 @@ def required_model_output_shape(
         Returns:
             size of the required input vector (minus leading batch dimension).
         """
+
+    @abc.abstractmethod
+    def get_state(self) -> dict:
```

Comment:
Dumb question: When would we need to store the state of a distribution? In case it contains (its own) model params?

Reply:
Maybe just mention an example for such a distribution here.

Reply:
Ah, got it. So the `inputs` (dist. parametrization) are already a distributions state. Still can you explain, why we would need this get/set_state infrastructure? Normally, distributions are super volatile and only exist for the time of a a single inference call.

Reply:
yep you are right. I was experimenting with different ways of passing back distribution objects from rolloutworkers to the learner actor. It turned out the dictionary of distribution parameters was sufficient and it worked. So I can get rid of this get/set_state functionality. No one really want to checkpoint a distribution.